q2_diversity/_beta/_pipeline.py

```diff
@@ -0,0 +1,68 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2020, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from q2_diversity_lib.beta import METRICS
+from q2_diversity_lib import translate_metric_name
+
+all_phylo_metrics = METRICS['PHYLO']['IMPL'] | METRICS['PHYLO']['UNIMPL']
+all_nonphylo_metrics = METRICS['NONPHYLO']['IMPL'] \
+                       | METRICS['NONPHYLO']['UNIMPL']
+metric_name_translations = METRICS['METRIC_NAME_TRANSLATIONS']
+
+
+def all_metrics():
+    return all_phylo_metrics | all_nonphylo_metrics
+
+
+def beta_phylogenetic(ctx, table, phylogeny,
+                      metric, threads=1,
+                      variance_adjusted=False,
+                      alpha=None,
+                      bypass_tips=False):
```

Comment:
```suggestion
def beta_phylogenetic(ctx,
                      table,
                      phylogeny,
                      metric,
                      threads=1,
                      variance_adjusted=False,
                      alpha=None,
                      bypass_tips=False):
```

might as well get everyone on their own lines, at this point.